src/pages/detail/tabs/intro/GroupIntroTab.tsx

```
@@ -7,14 +7,17 @@ import "katex/dist/katex.min.css";
 import "./styles.css";
 
 import useSWR from "swr";
-import { getNotionPage } from "../../apis/notion";
+
+import { getNotionPage } from "@/apis/notion";
 import NotionWrapper from "./NotionWrapper";
 
 interface GroupIntroTabProps {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)